src/tokenizer/index.js

```diff
@@ -591,16 +591,16 @@ export default class Tokenizer {
 
   // Read a string value, interpreting backslash-escapes.
 
-  readCodePoint() {
+  readCodePoint(throwOnInvalid) {
```

Comment:
@xtuc, I've tried to keep the style in this PR in keeping with the rest of babylon. I have no strong preference either way, but it seemed better to keep the style consistent for the moment. If maintainers have a particular style you'd like me to keep to I can go through and revise (and also revise my other PRs, maybe).